infrastructure/terraform/to_review/notify-service.tf

```diff
@@ -1,131 +0,0 @@
-resource "aws_sqs_queue" "notify_irc_queue" {
-  name = "${var.prefix}-notify-irc"
-}
-
-locals {
-  email_source_address = "${element(split("identity/", var.notify_ses_arn), 1)}"
-}
-
-module "notify_user" {
-  source = "modules/taskcluster-service-iam-user"
-  name   = "taskcluster-notify"
-  prefix = "${var.prefix}"
-
-  policy = <<EOF
-{
-    "Version": "2012-10-17",
-    "Statement": [
-        {
-            "Effect": "Allow",
-            "Action": [
-              "sqs:CreateQueue",
-              "sqs:GetQueueUrl",
-              "sqs:ReceiveMessage",
-              "sqs:SendMessage",
-              "sqs:DeleteMessage"
-            ],
-            "Resource": [
-              "${aws_sqs_queue.notify_irc_queue.arn}"
-            ]
-        },
-        {
-            "Effect": "Allow",
-            "Action": [
-                "ses:SendEmail",
-                "ses:SendRawEmail"
-            ],
-            "Resource": "*",
-            "Condition": {
-              "StringEquals": {
-                "ses:FromAddress": "${local.email_source_address}"
-              }
-            }
-        }
-    ]
-}
-EOF
-}
-
-module "notify_rabbitmq_user" {
```

Comment:
We need to bring this back.

Reply:
fixed at https://github.com/sciurus/taskcluster/pull/14/files#diff-fbb1294efb073c6a507ffdd613837248R5